modules/components/forms/text_input.cjsx

```diff
@@ -21,6 +21,7 @@ module.exports = class TextInput extends RestFormElement
 
   @defaultProps =
     inputType: 'text'
+    value:     ''
```

Comment:
React treats a `null` value as an uncontrolled input and throws a warning if you try to assign it a new value. They actually suggest defaulting it to `''` if you intend to change it.